src/CMakeLists.txt

```diff
@@ -187,6 +187,10 @@ target_link_libraries(simulator_navigation_demo
   simulator_base simulator_nav simulator_world_interface)
 target_link_libraries(simulator_navigation_demo ${sfml_libs})
 
+add_executable(gpsd_test
+  gps/gpsd_test.cpp)
+target_link_libraries(gpsd_test gps)
+
```

Comment:
For a lot of the testing programs I've been excluding them from the main target, so when we build the Rover program it isn't building all the test programs as well. Do you think this is a good policy, and should we implement it for this one? I also forgot to bring this up on the last pull request with the PID tuning program. 

If we want to implement that here, we can just add `set_property(TARGET gpsd_test PROPERTY EXCLUDE_FROM_ALL true)`.

Reply:
Hmm. I think we can potentially adopt the policy that people usually want to build individual targets (e.g. `make Rover`). So it's probably okay for `make` to by default build all of the targets---since most of the time people won't be doing that. In fact, we probably want cmake to build everything by default, because that's what we use for CI:

```
cmake --build ./build -j$(nproc)
./build/tests
```

We want to make sure CI is trying to build everything, so that we catch build errors when they're introduced.